extensions/wikia/AdEngine/js/config/desktop.js

```diff
@@ -5,6 +5,7 @@ define('ext.wikia.adEngine.config.desktop', [
 	'wikia.window',
 	'wikia.instantGlobals',
 	'wikia.geo',
+	'wikia.trackingOptOut',
```

Comment:
This guys is [deprecated](https://github.com/Wikia/app/blob/9d5242d7f6ef4e7a2a522276edbc99147205c4cd/resources/wikia/modules/trackingOptOut.js#L37). Use [`wikia.trackingOptIn`](https://github.com/Wikia/app/blob/9d5242d7f6ef4e7a2a522276edbc99147205c4cd/extensions/wikia/TrackingOptIn/js/trackingOptIn.js#L38) instead, please.